pkg/resource/deploy/step.go

```diff
@@ -730,7 +742,19 @@ func (s *ReadStep) Apply(preview bool) (resource.Status, StepCompleteFunc, error
 		s.old.Delete = true
 	}
 
-	complete := func() { s.event.Done(&ReadResult{State: s.new}) }
+	complete := func() {
+		now := time.Now().UTC()
+		s.new.Updated = &now
```

Comment:
Does a read count as an update? Kinda feel maybe this should just be nil as well.